src/helpers/LinkButtonComponent.types.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * The proptypes for {@link LinkButtonComponent}
+ */
+export interface LinkButtonComponentProps {
+  id?: string;
+  className?: string;
+  isStartButton?: boolean;
```

Comment:
Is this appropriate to be passed in to the user's component? Feels like that should be handled by the class we apply to it.

Reply:
If it _is_ appropriate to keep, it needs a comment explaining what it's for.